app/controllers/SeriesDetailsController.scala

```diff
@@ -79,6 +79,9 @@ class SeriesDetailsController @Inject() (
               .uncache()
           )
         case _ =>
+          if (!seriesStatus.contains(InProgressValue.value)) {
```

Comment:
Looks good, can you please add a unit test to verify if the `seriesStatus` is `None` then it should add a consignment status and if it is in progress then it shouldn't.

Reply:
As suggested I've added unit tests to cover those scenarios. I had to move the logic into its own function to make the testing a bit more manageable.